front-end/src/types/util.ts

```diff
@@ -0,0 +1 @@
+export type Optional<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
```

Comment:
```suggestion
export type Optional<T extends object, K extends keyof T = keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
```

1. T가 객체임을 명확히 하면 좋을거 같아요.
2. K의 디폴트 파라미터로 keyof T를 줘서, K가 입력되지 않았을때는 Partial 과 동일하게 쓸 수 있게 하면 어떨까요?



https://github.com/piotrwitek/utility-types/blob/df2502ef504c4ba8bd9de81a45baef112b7921d0/src/mapped-types.ts#L539

Reply:
2번은 그냥 레퍼런스보고 따라한거긴 한데, Partial 이랑 기능이 똑같으니 굳이 필요한가 싶긴 하네요 ㅎㅎ

Reply:
`T`에 대해서는 `객체`임을 한정 시키는 것이 좋겠네요 👍

Reply:
1번만 적용 :+1: